### ISSUE TITLE
can't save with original filename

### ISSUE THREAD

**Issue by mehmedean** [mpatric/mp3agic#123]
whenever I want to save edited mp3 with source name, it throws IllegalArgumentException exception. I can see that there is a check at the beginning of save() function. I think we should be able to update an existing mp3 file, via save() or an update() function.

**Comment by gameshack** [comment]
Agreed. I basically save my file twice now for that reason. First when i get it from the server and then again when tagging. After that i delete the original one. Would be great to be able to update existing files. 

See my "byte[]" post --> https://github.com/mpatric/mp3agic/issues/117

**Comment by albsch** [comment]
I agree. The library advertises manipulating mp3 tags to mp3 files and should be able to update existing files. Contrary to the mp3agic author's beliefs, it does prevent me from using the library. I do not want my code base to grow and increase in complexity to implement this basic feature.

**Comment by aauradnik** [comment]
If you really cannot work with this solution you could try the forked version jid3lib.
Save is handled much smoother there.
https://github.com/ericfarng/jid3lib
